packages/genesys-spark-components/.stylelintrc.json

```diff
@@ -6,7 +6,8 @@
         "src/components/stable/gux-accordion/**/*.scss",
         "src/components/stable/gux-button/**/*.{scss,html}",
         "src/components/stable/gux-button-slot/**/*.{scss,html}",
-        "src/components/beta/**/*.scss"
+        "src/components/beta/**/*.scss",
+        "src/components/stable/gux-[a-m]**/*.scss"
```

Comment:
Do you want to include the html files here too or have them a seperate PR?

Reply:
I'll come back for the html files. I'd rather get all the scss done first